reference-architecture/osp-cli/ch5.3_tenant_network.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+openstack network create tenant-network
+openstack subnet create --network tenant-network \
+  --subnet-range 172.18.20.0/24 --gateway none tenant-subnet
```

Comment:
I'd say to avoid hardcoding the network IPs and create variables with default values instead